app/src/main/java/me/saket/dank/ui/media/gfycat/GfycatRepositoryData.java

```diff
@@ -23,8 +23,8 @@
 
   @Inject
   public GfycatRepositoryData(@Named("gfycat_repository") RxSharedPreferences kvStore) {
-    needsAccessTokenHeaderStore = kvStore.getBoolean("gfycat_needs_access_token", false);
-    accessTokenExpiryTimeMillisStore = kvStore.getLong("gfycat_access_token_expiry_time_millis", currentTimeMillis());
+    needsAccessTokenHeaderStore = kvStore.getBoolean("gfycat_needs_access_token2", false);
```

Comment:
I've renamed this parameter in case if some users have it enabled now due to incorrect url errors